package.json

```diff
@@ -82,25 +82,25 @@
     "@netlify/local-functions-proxy": "1.1.1",
     "@netlify/redirect-parser": "14.5.0",
     "@netlify/zip-it-and-ship-it": "9.42.4",
-    "@octokit/rest": "20.1.1",
+    "@octokit/rest": "21.1.0",
     "@opentelemetry/api": "1.8.0",
     "ansi-escapes": "7.0.0",
     "ansi-to-html": "0.7.2",
     "ascii-table": "0.0.9",
     "backoff": "2.5.0",
     "better-opn": "3.0.2",
-    "boxen": "7.1.1",
+    "boxen": "8.0.1",
     "chalk": "5.4.1",
     "chokidar": "3.6.0",
     "ci-info": "4.1.0",
     "clean-deep": "3.4.0",
     "commander": "10.0.1",
     "comment-json": "4.2.5",
-    "configstore": "6.0.0",
+    "configstore": "7.0.0",
```

Comment:
now requires node 18 (we already do): https://github.com/sindresorhus/configstore/releases/tag/v7.0.0